starter-code/public/admin.html

```diff
@@ -39,6 +45,14 @@
         <p><em>Details on who is writing, and how much writing they are doing...</em></p>
         <ul class="author-stats">
         </ul>
+
```

Comment:
Make sure to remove commented code, prior to pushing to github. Just good practices to get used to.